sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/ResourceManagerUtils.java

```diff
@@ -26,7 +26,13 @@
  * Defines a few utilities.
  */
 public final class ResourceManagerUtils {
-    private ResourceManagerUtils() {
+    /**
+     * Creates a new instance of ResourceManagerUtils.
+     *
+     * @deprecated ResourceManagerUtils is a final class.
+     */
+    @Deprecated
+    public ResourceManagerUtils() {
```

Comment:
private, not `@Deprecated`

Reply:
private

Reply:
Fixed in the c[ommit#7aef3c2](https://github.com/Azure/azure-sdk-for-java/pull/43192/commits/7aef3c200b307b9e558e92b532e228d1141a193e)